powershell.ps1

```diff
@@ -0,0 +1,3 @@
+cd C:\Users\[YOUR-USER]\chiadog
```

Comment:
Can we automate this as well? See https://stackoverflow.com/a/2085755

Reply:
It's certainly possible.  In reality, the user can place the directory anywhere on-disk.  To be accurate, it should be changed to "cd <path>" but according to your instructions, you used this placeholder notation to correlate with your "quick guide" instructions.  It's rather easy to get the username for every platform, but I assume you hadn't automated that since a savvy user would immediately read between the lines to know "put your desired path here".

If you want to hard-code chiadog to the metaphorical "~/chiadog" on every platform, then it probably makes sense to merge this as is, then do an all-encompassing commit to do an automated home-dir-path on every platform.  I personally think it goes a bit too far, but hey... it's not my project :)   If you want me to automate hard-coding the homedir to `%userprofile%/chiadog` so that you can handle every other platform shortly after, that's fine.  Lemme know !

Reply:
with just forcing the referenced path to the platform-specific directory and keying off the CWD for the launch stub (`start.cmd`), we actually don't need the user to hand-jam the powershell launch script 👍 

I've created the "scripts" dir for platform-specific stuff and made the discussed changes

Reply:
Perfect, thanks for thinking that through and resolving it in a meaningful way. I didn't consider the full context with the initial suggestion but you understood the idea. 👍